profiler/src/ProfilerEngine/Datadog.Profiler.Native/CorProfilerCallback.cpp

```diff
@@ -794,6 +792,23 @@ void CorProfilerCallback::ConfigureDebugLog()
     }
 }
 
+#define PRINT_ENV_VAR_IF_SET(name)                            \
+    {                                                         \
+        auto envVarValue = shared::GetEnvironmentValue(name); \
+        if (!envVarValue.empty())                             \
+        {                                                     \
+            Log::Info("  ", name, ": ", envVarValue);         \
+        }                                                     \
+    }
+
+void CorProfilerCallback::PrintEnvironmentVariables()
+{
+    // TODO: add more env vars values
```

Comment:
This is a good idea: we should also provide the command line like the Tracer is doing